### PR TITLE
Custom MultiAddress

### LINKED ISSUE #1232: Can we lookup ethereum address for <T::Lookup as StaticLookup>::Source
For example, someone directly use Ethereum address as the dest in "Balance.transfer".

https://crab.subscan.io/extrinsic/8807384-1

If <T::Lookup as StaticLookup>::Source can helo lookup the ethereum address, this might be also possible, and more user friendly.

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Close #1232.

image
image
image

### CODE REVIEW

**Review by AsceticBear — APPROVED** [review]
:+1:

**Comment by AurevoirXavier** [comment]
image

cc @JayJay1024

File: node/runtime/pangolin/src/pallets/system.rs [hunk 1]
```diff
@@ -69,7 +39,7 @@ impl Config for Runtime {
 	type Hashing = Hashing;
 	type Header = Header;
 	type Index = Nonce;
-	type Lookup = DarwiniaAccountLookup<AccountId>;
+	type Lookup = DarwiniaAccountLookup;
```
Comment (AsceticBear) [inline node/runtime/pangolin/src/pallets/system.rs:42]: Please update the pangoro runtime as well.

File: node/runtime/template/src/pallets/system.rs [hunk 2]
```diff
@@ -26,7 +26,7 @@ impl Config for Runtime {
 	type Hashing = Hashing;
 	type Header = Header;
 	type Index = Nonce;
-	type Lookup = AccountIdLookup<AccountId, ()>;
+	type Lookup = DarwiniaAccountLookup;
```
Comment (AsceticBear) [inline node/runtime/template/src/pallets/system.rs:29]: Delete `use sp_runtime::traits::AccountIdLookup;`

**Pull request merged**